magefile.publish.go

```diff
@@ -0,0 +1,137 @@
+//go:build mage
+// +build mage
```

Comment:
`go:build` [is enough](https://stackoverflow.com/a/68361186)

Reply:
Fixed